dts/bindings/clock/fixed-clock.yaml

```diff
@@ -21,10 +21,5 @@ properties:
       description: output clock frequency (Hz)
       required: true
 
-    clocks:
```

Comment:
Is this supposed to be removed?

Reply:
Yup, forgot to explain it. See the updated commit message for the commit that also has the code changes (it needs to go in with that one).

Guessing it was supposed to be `type: phandle-array` (currently you'd have to assign it like `clocks = <1 2 3>`, whatever that means), and it's already in `base.yaml`. Probably close enough in meaning too.